less/styles.less

```diff
@@ -132,4 +164,15 @@
   img{
     margin-top:45px;
   }
+}
+
+.founder-logo {
+  filter: grayscale(1) contrast(0)
```

Comment:
I am pretty sure this is not allowed if we take a look at the logo guidelines of each member. I would recommend that we remove the grayscale of each logo.

Reply:
Really? This has become pretty standard fair on modern websites. Here's an example of (black this time) treatment on logos at Hashicorp's site:

https://www.hashicorp.com/

We can remove if it's a blocker, but certain is aesthetically more pleasing within the design ;)

Reply:
+1 I agree this is way nicer but it's a blocker. Altering a company logo is not something that someone should do without proper permission.

Most brand usage guideline will require you to use their grayscale version of their logo.

Reply:
We should ask our members to supply a grayscale version of their logo if we need it on the website!

Reply:
ok